tutorials/cv-package-aicore-train-object-detection/cv-package-aicore-train-object-detection.md

```diff
@@ -331,45 +347,43 @@ except IndexError:
         name=config_name,
         scenario_id=workflow_config["labels"]["scenarios.ai.sap.com/id"],
         executable_id=workflow_config["name"],
-        input_artifact_bindings=[InputArtifactBinding('datain', artifact.id)],
-        parameter_bindings=params
+        input_artifact_bindings=[InputArtifactBinding('datain', artifact.id)], # dataset to use `datain` is placeholder name in the template
+        parameter_bindings=params # hyper-parameter values to use.
     )
 ```
 
-The above snippet checks if a configuration of the same name already exists in AI Core. If it exists, it will use the existing configuration.
 
 !![image](img/create-configuration.png)
 
-If it does not exist, it will create a new configuration using the following details:
-
-- `name` : Name of configuration used to recognize it
+### Details of configuration variables
 
-- `scenario_id`: Connect the configuration to the scenario id mentioned previously in our workflow configuration.
-
-- `executable_id`: Name of the workflow mentioned in workflow configuration.
-
-- `input_artifact_binding` - Binds the dataset artifact you created in previous tutorial and binds it as an Input Artifact for this workflow. [TODO: Explain succinctly]
-
-- `parameter_bindings` - Parameters for our Machine learning training you set in previous step.
+| Parameter | Description |
+| --- | --- |
+| `name` | Name of configuration |
+| `scenario_id` | Used to connect the configuration to the scenario ID mentioned previously in our workflow configuration |
```

Comment:
Please don't say "we", "us", "our", etc. unless you are referring to SAP.

Reply:
done